### PR TITLE
PLANET-4182 Columns block images are not set in block back end

### PR DESCRIPTION
**Pull request by dpivo**
- Added imageOrButton function to display image if one has been selected.

### CODE REVIEW

File: react-blocks/build/editorStyle.min.css.map [hunk 1]
```diff
@@ -0,0 +1 @@
...rousel-header .carousel-control-next {\n    width: 50%;\n    margin-right: -25%; } }\n\n.splittwocolumns-block-issue-imgs, .splittwocolumns-block-tag_imgs {\n  width: 250px;\n  padding: 10px 10px; }\n\n.submenu-block-attribute-wrapper {\n  display: inline-block;\n  margin-right: 20px; }\n\n.submenu-level-link {\n  display: inline-block;\n  margin-right: 20px; }\n  .submenu-level-link .components-base-control__field {\n    margin-bottom: 13px; }\n\n.LayoutSelector {\n  display: flex;\n  width: 100%;\n  margin-bottom: 30px;\n  cursor: pointer; }\n  .LayoutSelector .LayoutOption {\n    padding: 20px;\n    border-right: 1px solid #ccc;\n    width: 100%; }\n  .LayoutSelector .LayoutOption:last-child {\n    border-right: 0; }\n  .LayoutSelector .help {\n    font-family: \"Roboto\", sans-serif;\n    font-size: 10pt;\n    margin-top: 10px; }\n\n.EmptyMessage {\n  font-family: \"Roboto\", sans-serif;\n  padding: 32px;\n  background: #e3f2d3; }\n\n.Preview .PreviewBar {\n  border-bottom: 1px solid #8bcc49;\n  font-style: italic;\n  padding: 12px; }\n\n.Preview .submenu-sidebar {\n  float: unset !important; }\n\n.Preview .social-media-embed {\n  float: unset !important; }\n"],"sourceRoot":""}
```
Comment (kirdia) [inline react-blocks/build/editorStyle.min.css.map]: this is before we rename the react-blocks to assets

**Comment by kirdia** [comment]
Needs cleanup. Includes files from react-blocks folder.
Build files contain merge comments
`<<<<<<< HEAD:assets/build/editorIndex.js`

**Comment by dpivo** [comment]
> Needs cleanup. Includes files from react-blocks folder.
> Build files contain merge comments
> `<<<<<<< HEAD:assets/build/editorIndex.js`

@kirdia Cleared up, Can you have another look?

**Comment by kirdia** [comment]
Looks better!. It needs rebase over develop

**Comment by dpivo** [comment]
> Looks better!. It needs rebase over develop

@kirdia  done 👍

**Pull request merged**